ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1288 @@
+{
```

Comment:
<p>There are ten more ways to get Imagenette2-320 dataset, why mention only one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>